charts/spire/charts/tornjak-frontend/templates/tests/test-tornjak-connection.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "tornjak-frontend.fullname" . }}-test-connection"
+  namespace: {{ include "tornjak-frontend.namespace" . }}
+  labels:
+  annotations:
+    "helm.sh/hook": test
+spec:
+  securityContext:
+    {{- toYaml .Values.podSecurityContext | nindent 4 }}
+  containers:
+    - name: curl-tornjak-frontend
+      image: cgr.dev/chainguard/bash:latest
+      command: ['curl']
+      args: ['-k', '-s', '-f', 'http://{{ include "tornjak-frontend.fullname" . }}.{{ include "tornjak-frontend.namespace" . }}.svc.cluster.local:{{ .Values.service.port }}']
```

Comment:
@kfox1111 what was again the new helper function for clusterdomain?

Reply:
This has to be addressed, otherwise we reintroduce a bug.

Reply:
see https://github.com/spiffe/helm-charts/pull/179/files#diff-7778c29eb75aa21d8e4de97c8a8a5920d6a81f28571aa258c258695b943d4017R18 for a reference